src/gobexport/meetbouten/types.py

```diff
@@ -25,11 +25,11 @@ def _to_string(value):
     :return:
     """
     assert(type(value) is str or value is None)
-    value = "" if value is None else str(value).replace("\r", "").replace("\n", " ")
-    return f'$${value}$$'
+    value = '' if value is None or value == '' else str(f'$${value}$$').replace("\r", "").replace("\n", " ")
```

Comment:
This makes the to_string method quite useless for any other conversions.
But I do not see any other possibility for now...